dashboard/package.json

```diff
@@ -6,9 +6,9 @@
     "@fortawesome/fontawesome-svg-core": "^6.1.1",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/react-fontawesome": "^0.1.18",
-    "@patternfly/patternfly": "^4.183.1",
-    "@patternfly/react-core": "^4.198.19",
-    "@patternfly/react-table": "^4.75.2",
+    "@patternfly/patternfly": "4.224.5",
+    "@patternfly/react-core": "4.198.19",
+    "@patternfly/react-table": "4.75.2",
```

Comment:
So 4.225 had a breaking change? Locking to an old version can be a great quick fix, but how much work would be required to work on the latest v4.x? Locking to old versions always brings a risk of missing security updates, for example.